### ISSUE TITLE
The accuracy on cifar10 dataset

### ISSUE THREAD

**Issue by deloresmorgen** [KarhouTam/FedRecon#2]
why my training accuracy on cifar10 dataset is low? The training hyperparameters are as follows: 'algo': 'fedrecon', 'global_epochs': 100, 'pers_epochs': 1, 'recon_epochs': 1, 'pers_lr': 0.005, 'recon_lr': 0.01, 'server_lr': 1.0, ' client_num_per_round': 5, 'dataset': 'cifar10', 'no_split': 0, 'eval_while_training': 1, 'seed': 17, 'batch_size': 32, 'gpu': 1, 'valset_ratio'. 0.1, 'log': 0, the accuracy of training with this parameter is only 25%-30%

**Comment by KarhouTam** [comment]
Sorry for my late answer. This repo is not maintained for a long time and I really forget about the code details. So I can't explain to your experiment result. 🙏
